server/routes/creatingLicences/handlers/checkAnswers.ts

```diff
@@ -34,6 +34,7 @@ export default class CheckAnswersRoutes {
 
     const conditionsToDisplay = await this.conditionService.getAdditionalAPConditionsForSummaryAndPdf(licence, user)
     const bespokeConditionsToDisplay = await this.conditionService.getbespokeConditionsForSummaryAndPdf(licence, user)
+    const omuEmail = (await this.licenceService.getOmuEmail(licence.prisonCode, user))?.email
```

Comment:
isn't the error on the kotlin side, I thought you'd get a 500 response?

Reply:
The usual error is when the OMU email is missing which I thought returned a 404, but now I'm doubting myself so will give it another test

Regardless, this should handle a 500 as `getOmuEmail` returns `null` for >=400 and < 500, or the error if it's outside of that range
In either case, <response>?.email should return null, and thereby be handled, no?